src/openforms/scss/components/admin/_json-dump-variables.scss

```
@@ -0,0 +1,8 @@
+.json-dump-variables {
+  display: flex;
+
+  &--horizontal {
```

For new styling it would be nice to use the scss bem helper functions

```suggestion
@include bem.modifier('horizontal') {
```